docs/source/installation.rst

```diff
@@ -3,43 +3,114 @@
 Installation
 ============
 
-sktime is available via PyPI using:
+``sktime`` requires an environment with python version 3.6 or higher.
+
+Release versions
+----------------
+
+For frequent issues with installation, consult the `Release versions - troubleshooting`_ section.
+
+Installing sktime from PyPI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``sktime`` releases are available via PyPI and can be installed via ``pip`` using:
 
 .. code-block:: bash
 
     pip install sktime
 
-But note that the package is actively being developed and currently not feature stable.
+This will install ``sktime`` with core dependencies, excluding interfaced packages.
 
+To install ``sktime`` with maximum dependencies, including all interfaced packages, install with the ``all_extras`` modifier:
```

Comment:
oh, it's not? So, not all interfaced packages are soft depdenies? Or not all soft dependencies are coming from interfaced packages?

Where I understand soft dependency in this context as something where the *framework* works without problems, but adapters may not. Is this understanding correct?

Reply:
Yes, for example, statsmodels is a hard dependency, mainly because we use it in multiple places in sktime. Other packages like tsfresh or PyOD are soft dependencies, they're only required for using specific modules. There is also a small paragraph on this here: https://github.com/alan-turing-institute/sktime/blob/main/CONTRIBUTING.md#dependencies

Reply:
I've updated the paragraph.